CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace CanHazFunny;
+public class Jester
+{
+    private Comedian? _Comedian;
+    private JokeService? _JokeService;
+    private string? _Joke;
+    public Comedian? Comedian 
+    {
+        get => _Comedian!;
+        set 
+        { 
+            if (value is null) throw new ArgumentNullException(nameof(value));
+            _Comedian = value;
+        } 
+    }
+    public JokeService? JokeService
+    {
+        get => _JokeService!;
+        set
+        {
+            if (value is null) throw new ArgumentNullException(nameof(value));
+            _JokeService = value;
+        }
+    }
+
+    public string?Joke
+    {
+        get => _Joke!;
+        private set 
+        {
+            if (value is null) throw new ArgumentNullException(nameof(value));
+            _Joke = value; 
+        }
+    }
+
+    public Jester()
+    {
+        Joke = "Chuck Norris";
+    }
```

Comment:
This should take IJokeService and IComedian as dependencies and null check them.
```suggestion
    public Jester(IJokeable jokeable, IComedian comedian)
    {
        Jokeable = jokeable ?? throw new ArgumentNullException(nameof(jokeable));
        Comedian = comedian ?? throw new ArgumentNullException(nameof(comedian));
    }
```